src/components/brew-input.js

```diff
@@ -1,29 +1,33 @@
 import React from 'react';
 import {connect} from 'react-redux';
+import moment from 'moment';
 
 import {addBrew} from '../database/brew';
 import {FancyInput, FancyDropdown, FancyButton} from './input-stuff.js'
 
 
 var testEquip = ["Chemex", "Clever", "French Press"];
-// Eventually this would be stored elsewhere, yes?
-
 
 const FirstPanel = (props) => {
   return (
     <div>
       <FancyInput label="Coffee Name " name="name" handleChange={props.handleChange}
         value={props.name}/>
-      <FancyDropdown label="Brew Method " name="brewMethod" options={testEquip}
-        handleChange={props.handleChange}/>
+      <FancyDropdown label="Brew Method " name="brewMethod" handleChange={props.handleChange}
+        options={testEquip}
+        />
       <FancyInput label="Brew Date" name="brewDate" handleChange={props.handleChange}
-        value ={props.brewDate} />
+        value={props.brewDate} />
+      <FancyButton name="dateButton"
+        handleSubmit={props.setDateToday}
```

Comment:
can do this later, but i would avoid calling this prop `handleSubmit` and instead calling it just `onClick` so that it's clear that you're just proxying the handler to the button for the `click` event and not anything to do with the `submit` event.